src/App/App.test.js

```diff
@@ -1,9 +1,40 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { BrowserRouter, Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
 import { App } from './App';
+import ApiCalls from '../ApiCalls.js';
+import userEvent from '@testing-library/user-event';
+jest.mock('../ApiCalls.js');
+
 
 describe('App', () => {
```

Comment:
I'm a little unclear on this, but should all of our integration tests belong in the App test? It feels a bit sparse to me, but I could be wrong. Might want to get a second opinion on this.

Reply:
I completely agree. I am getting the feeling that maybe all integrations tests belong in App? We should definitely seek some clarification on this from our instructors & refactor if we have time before the due date.